io/driver/driver_parameter_block_device.py

```diff
@@ -104,6 +105,7 @@ def is_mpath_flushed(self):
         '''
         process.system("multipath -F", ignore_status=True)
         cmd = "lsmod | grep -i ^%s" % self.module
+        time.sleep(10)
```

Comment:
@maramsmurthy we need to do more debugs for this with dev guys like why the mutipath reset is happening after it is flushed properly. but currently we can use this work around as, if the driver is unloaded and test fails all other rest of the test cases will fail and it breaks the chain.
So lets pursue this for now as work around. please add a to-do note, so that it can be enhanced in better way in future.

Reply:
Sure Naresh! Will add todo note and update you